.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,11 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+    tags:
+
+jobs:
+  ci:
+    uses: laminas/workflow-continuous-integration/.github/workflows/continuous-integration.yml@1.x
```

Comment:
This didn't run anything - perhaps not applicable?

Reply:
no, I believe it didn't run because this workflow does not exist on the default branch.

Runs on fork https://github.com/ghostwriter/laminas-org-laminas.dev/runs/5278509505